### PR TITLE
ls getVarIds now search tags too

### PR DESCRIPTION
**Pull request by dehann**

### CODE REVIEW

**Comment by dehann** [comment]
still working on sorting the ls return list (currently affects plotting)

File: src/services/GraphsDFG.jl [hunk 1]
```diff
@@ -342,6 +361,83 @@ Checks if the graph is not fully connected, returns true if it is not contiguous
...
+  while !allnums(st[i:i])  i+=1; end
+  return i
+end
+
+"""
+    $SIGNATURES
+
+Sort a variable list which may have nested structure such as `:x1_2` -- does not sort for alphabetic characters.
+"""
+function sortVarNested(vars::Vector{Symbol})::Vector{Symbol}
```
Comment (GearsAD) [inline src/services/GraphsDFG.jl]: Is this used in GraphsDFG? If not, do you mind it to a common utils file?

Comment (dehann) [reply src/services/GraphsDFG.jl]: think that should be better?

**Comment by dehann** [comment]
solves #24

**Pull request merged**